test/SourceKit/InterfaceGen/gen_clang_module.swift.apinotes_swift3.response

```diff
@@ -0,0 +1,1232 @@
+import Foundation
+
+
+open class Foo : NSObject {
+}
+
+open class GlobalToMember_Class_Container : NSObject {
+}
+public typealias GlobalToMember_Class_Payload = GlobalToMember_Class_Container.Payload
+extension GlobalToMember_Class_Container {
+
+    open class Payload : NSObject {
+    }
+}
+
+open class MemberToGlobal_Class_Container : NSObject {
+}
+open class MemberToGlobal_Class_Payload : NSObject {
+}
+extension MemberToGlobal_Class_Container {
+
+    public typealias Payload = MemberToGlobal_Class_Payload
+}
+
+open class MemberToMember_Class_Swift3 : NSObject {
+}
+open class MemberToMember_Class_Swift4 : NSObject {
+}
+extension MemberToMember_Class_Swift3 {
+
+    open class Payload : NSObject {
```

Comment:
This is incorrect. I would expect a typedef here that points to the real name that ended up getting used, MemberToMember_Class_Swift4.Payload. (And the same for the other member-to-member test).

Reply:
Yeah, I agree. I expect:
```swift
extension MemberToMember_Class_Swift3 {
    typealias Payload = MemberToMember_Class_Swift4.Payload
}
extension MemberToMember_Class_Swift4 {
    open class Payload : NSObject {
    }
}
```
I'm investigating. But I think it's possible that ImportDecl doesn't do the right thing, because my change does not  manipulate the AST hierarchy.

Reply:
Yep, definitely possible!